packages/docusaurus-theme-classic/src/theme/BlogPostItem/index.tsx

```diff
@@ -129,7 +129,18 @@ function BlogPostItem(props: Props): JSX.Element {
 
   return (
     <>
-      <Seo {...{keywords, image}} />
+      {isBlogPostPage ? (
+        <Seo {...{keywords, image}}>
+          <meta property="og:type" content="article" />
+          <meta property="article:published_time" content={date} />
+          {authorURL && <meta property="article:author" content={authorURL} />}
+          {frontMatter.tags && (
+            <meta property="article:tag" content={frontMatter.tags.join(',')} />
+          )}
+        </Seo>
+      ) : (
+        <Seo {...{keywords, image}} />
```

Comment:
I think we shouldn't need that either and is probably a mistake?

It doesn't make sense to me to render the meta keywords of the last blog post of the list 🤪

Reply:
What about just moving this Seo comp to BlogPostPage?

The list should have totally different metadata than a single blog post imho

Reply:
So I wanted SEO to continue doing what it did before... Are you saying you'd like to remove the SEO component entirely from this page?  As I didn't add it there; it was already present.

Reply:
The change as implemented right now renders the SEO component as it already is in the case that this is *not* a `isBlogPostPage`, and in the case where it is, it renders the SEO component with the child meta tags.

Reply:
So the current implementation is this:

```jsx
<Seo {...{keywords, image}} />
```

With the changes in this PR that will still be rendered in the case where `isBlogPostPage === false`

When it is true it will render:

```jsx
        <Seo {...{keywords, image}}>
          <meta property="og:type" content="article" />
          <meta property="article:published_time" content={date} />
          {authorURL && <meta property="article:author" content={authorURL} />}
          {frontMatter.tags && (
            <meta property="article:tag" content={frontMatter.tags.join(',')} />
          )}
        </Seo>
```

If we pull `Seo` entirely from this screen then presumably that breaks other use cases?

Reply:
I guess we could just put Seo back to how it was before and do this in `BlogPostPage`:

```
<Head>
          <meta property="og:type" content="article" />
          <meta property="article:published_time" content={date} />
          {authorURL && <meta property="article:author" content={authorURL} />}
          {frontMatter.tags && (
            <meta property="article:tag" content={frontMatter.tags.join(',')} />
          )}
</Head>
```

Would that work?

Reply:
Implemented this approach.  That what you had in mind?

Reply:
🤪 

> So I wanted SEO to continue doing what it did before... Are you saying you'd like to remove the SEO component entirely from this page?

Yes that's exactly what I mean: what we already do is incorrect and should be fixed, so this PRR is a good opportunity to do so :)


https://socialsharepreview.com/?url=https://docusaurus.io/blog

=> we should probably not display the social card + keywords of the last blog post of the list here.

Just move `Seo` to `BlogPostPage` instead.


It does not make much sense to me anyway to declare SEO metadata on reusable comps that can be useful in both list+details pages.

Reply:
Okay cool - I'm away from my keyboard the next couple of weeks. If you'd like to work on this branch in the meantime please feel free. If not I can take a look when I'm back!

Reply:
fixed :)